src/Moka/Exception/MissingDependencyException.php

```diff
@@ -0,0 +1,8 @@
+<?php
+declare(strict_types=1);
+
+namespace Moka\Exception;
+
+class MissingDependencyException extends \Exception
+{
```

Comment:
Here maybe you should move here the sprintf with the message that you have in the check...